src/languages/python.js

```diff
@@ -113,6 +130,14 @@ function(hljs) {
         className: 'meta',
         begin: /^[\t ]*@/, end: /$/
       },
+      {
+        className: 'type',
+        begin: /([\w]+)=[\w"'\.\x20]+[,)]/
+      },
```

Comment:
Please add a `markup` test that shows that both these rules (type and symbol) work as expect and provides and example.